packages/jokul/package.json

```diff
@@ -80,6 +84,7 @@
         "cssnano-preset-lite": "^3.0.0",
         "glob": "^11.0.0",
         "postcss": "^8.4.24",
+        "react-hook-form": "^7.44.3",
```

Comment:
Denne bør vel med i `optionalDependencies`?
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#optionaldependencies

Reply:
Den er bare brukt for å generere typer, så tror det holder at den er i devDependencies :)